src/app/projects/add-edit-project/add-edit-project.component.html

```diff
@@ -205,7 +205,7 @@ <h2>Engagement Information</h2>
         <div class="add-row med">
           <label for="enagementInfo">Engagement Tab Information </label>
           <ckeditor name="engagementInfo" id="engagementInfo" formControlName="engagementInfo" [editor]="Editor"
-            [data]="project?.engagementInfo"></ckeditor>
+            [data]="project?.engagementInfo" (ready)="editorOnReady($event)"></ckeditor>
```

Comment:
FYI @ShawnTurple I actually tested this and it looks like to access that event object, we _do_ need to prefix it with the dollar sign.